spec/game_play_spec.rb

```diff
@@ -40,4 +39,102 @@
         expect(@game_play.select_spot(@board)).to eq(9)
     end
   end
+
```

Comment:
Nice tests!!!  Very well organized, named clearly, and testing several cases - love to see it!

Reply:
Thank you!!